tests/src/Kernel/MultilingualBlockTest.php

```diff
@@ -92,76 +92,76 @@ public function testBlockRendering(): void {
     $button_header = $crawler->filter('button.btn-close');
     $this->assertSame('modal', $button_header->attr('data-bs-dismiss'));
     $link_language = $crawler->filter('a#link_bg');
-    $this->assertSame('http://localhost/bg/%3Cnone%3E', $link_language->attr('href'));
+    $this->assertEmpty($link_language->attr('href'));
```

Comment:
Quite some weird old assertions here, was it a big bug?

Reply:
It's that change in drupal core rout matching, current_route_match for this Kernel test used to be `<none>` and after the code change its just empty